<!DOCTYPE html> <html lang=""> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Molecular technology development | Hesselberth Lab</title> <meta name="author" content="Hesselberth Lab"> <meta name="description" content="Hesselberth Lab at the University of Colorado Asnchutz Medical Campus"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="preconnect" href="https://fonts.googleapis.com"> <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://hesselberthlab.org/projects/molecular-tech/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> <script defer data-domain="hesselberthlab.org" src="https://plausible.io/js/script.js"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Hesselberth </span>Lab</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/team/">team</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">projects</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/projects/rna-repair/">RNA damage and repair</a> <a class="dropdown-item" href="/projects/molecular-tech/">Molecular technologies</a> <a class="dropdown-item" href="/projects/rna-therapies/">RNA therapies</a> <a class="dropdown-item" href="/projects/software/">Software</a> </div> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/fun/">fun</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Molecular technology development</h1> <p class="post-description"></p> </header> <article> <h3 id="single-cell-biochemistry-a-new-approach-to-measure-functional-heterogeneity">Single-cell biochemistry: a new approach to measure functional heterogeneity</h3> <p>New methods to study heterogeneity at cellular resolution in complex tissues are transforming our understand-ing of human biology and disease. These approaches measure differences in gene expression, chromatin ac-cessibility, and protein levels across thousands to millions of cells to understand developmental trajectories of tissues, tumors, and whole organisms. But these methods rely on measurements of static levels of DNA, RNA, and proteins, and fail to capture dynamic biochemical activities that underlie complex cellular functions. Instead of developing more direct readouts of cellular function, the field has focused on inferring functional status from measurement of mRNA abundance and chromatin accessibility in single cells.</p> <p>To accelerate the study of biochemical heterogeneity among single cells, we are developing functional assays as a new modality for single-cell experiments. Instead of measuring the abundance of molecules – levels of DNA, RNA, or protein – from single cells and predicting cell functional states, our key innovation is to directly measure enzymatic activities in single cells by measuring the conversion of substrates to products by single cell extracts in a high-throughput DNA sequencing experiment. Our approach is compatible with existing platforms that measure gene expression in thousands to millions of individual cells and enables many different enzymatic activities to be measured simultaneously.</p> <div class="publications"> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/richer-umap-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/richer-umap-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/richer-umap-1400.webp"></source> <img src="/assets/img/publication_preview/richer-umap.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="richer-umap.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Richer2020-dx" class="col-sm-8"> <div class="title">Simultaneous measurement of biochemical phenotypes and gene expression in single cells</div> <div class="author"> Amanda L Richer, Kent A Riemondy, Lakotah Hardie, and Jay R Hesselberth</div> <div class="periodical"> <em>Nucleic Acids Res.</em>, Jun 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://pubmed.ncbi.nlm.nih.gov/32286626" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PubMed</a> <a href="https://github.com/hesselberthlab/sc-haircut" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">GitHub</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="32286626" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Methods to measure heterogeneity among cells are rapidly transforming our understanding of biology but are currently limited to molecular abundance measurements. We developed an approach to simultaneously measure biochemical activities and mRNA abundance in single cells to understand the heterogeneity of DNA repair activities across thousands of human lymphocytes, identifying known and novel cell-type-specific DNA repair phenotypes.</p> </div> </div> </div> </li></ol> </div> <h3 id="charting-tissue-dissmination-with-molecular-tracking-devices">Charting tissue dissmination with molecular tracking devices</h3> <p>The dissemination of molecules through tissue establishes signaling gradients that drive complex developmental patterns, enables rapid responses to infection, and dictates the distribution of therapeutic drugs. However, our understanding of this molecular cartography would be vastly improved by charting molecular traffic throughout organ systems and cataloguing cell and tissue responses upon arrival and sensing.</p> <p>We recently developed “molecular tracking devices” – protein-DNA conjugates that are stable during tissue transit <em>in vivo</em> – to follow the distribution, acquisition, and retention of antigen in the immune system during an immune response. Molecular tracking devices enable previously inaccessible measurements of gene expression responses to signaling gradients and ligand-receptor interactions in living animals. We are expanding this approach to study the dynamics and concentrations of tracking devices within tissue and associated gene expression responses caused by receptor activation.</p> <div class="publications"> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/sheridan-umap-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/sheridan-umap-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/sheridan-umap-1400.webp"></source> <img src="/assets/img/publication_preview/sheridan-umap.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="sheridan-umap.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Walsh2021-sh" class="col-sm-8"> <div class="title">Molecular tracking devices quantify antigen distribution and archiving in the murine lymph node</div> <div class="author"> Shannon M Walsh, Ryan M Sheridan, Erin D Lucas, Thu A Doan, Brian C Ware, Johnathon Schafer, Rui Fu, Matthew A Burchill, Jay R Hesselberth, and Beth Ann Jiron Tamburini</div> <div class="periodical"> <em>Elife</em>, Apr 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://pubmed.ncbi.nlm.nih.gov/33843587" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PubMed</a> <a href="https://www.biorxiv.org/content/10.1101/2020.08.19.219527" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">bioRxiv</a> <a href="https://github.com/rnabioco/antigen-tracking" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">GitHub</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="33843587" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The detection of foreign antigens in vivo has relied on fluorescent conjugation or indirect read-outs such as antigen presentation. In our studies, we found that these widely used techniques had several technical limitations that have precluded a complete picture of antigen trafficking or retention across lymph node cell types. To address these limitations, we developed a 'molecular tracking device' to follow the distribution, acquisition, and retention of antigen in the lymph node. Utilizing an antigen conjugated to a nuclease-resistant DNA tag, acting as a combined antigen-adjuvant conjugate, and single-cell mRNA sequencing, we quantified antigen abundance in the lymph node. Variable antigen levels enabled the identification of caveolar endocytosis as a mechanism of antigen acquisition or retention in lymphatic endothelial cells. Thus, these molecular tracking devices enable new approaches to study dynamic tissue dissemination of antigen-adjuvant conjugates and identify new mechanisms of antigen acquisition and retention at cellular resolution in vivo.</p> </div> </div> </div> </li></ol> </div> <h3 id="funding">Funding</h3> <ul> <li> <a href="https://research.cuanschutz.edu/research-ppg/cu-anschutz-som-programmatic-incubator-for-research-(cu-aspire)-program" rel="external nofollow noopener" target="_blank">CU ASPIRE</a> (with Beth Tamburini, Tem Morrison, and Jenna Guthmiller)</li> <li><a href="https://reporter.nih.gov/search/JVJbnKPKPEmsn4iajmiqig/project-details/10263944" rel="external nofollow noopener" target="_blank">NIH/NIA Transformative R01</a></li> <li><a href="https://reporter.nih.gov/search/HH46I88QtUOO7kXMtKO1fg/project-details/10105545" rel="external nofollow noopener" target="_blank">NIH/NIAID R21</a></li> </ul> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2024 Hesselberth Lab. Powered by <a href="http://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Last updated: July 02, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js"></script> <script defer src="/assets/js/common.js"></script> <script defer src="/assets/js/copy_code.js" type="text/javascript"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>