<!DOCTYPE html> <html lang=""> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Hesselberth Lab</title> <meta name="author" content="Hesselberth Lab"> <meta name="description" content="Hesselberth Lab at the University of Colorado Asnchutz Medical Campus"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="preconnect" href="https://fonts.googleapis.com"> <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://hesselberthlab.org/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> <script defer data-domain="hesselberthlab.org" src="https://plausible.io/js/script.js"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <div class="navbar-brand social"> <a href="https://github.com/rnabioco" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fab fa-github"></i></a> <a href="https://scholar.google.com/citations?user=EnOmNEYAAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://twitter.com/jayhesselberth" title="Twitter" rel="external nofollow noopener" target="_blank"><i class="fab fa-twitter"></i></a> </div> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/team/">team</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">projects</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/projects/rna-repair/">RNA damage and repair</a> <a class="dropdown-item" href="/projects/molecular-tech/">Molecular technologies</a> <a class="dropdown-item" href="/projects/rna-therapies/">RNA therapies</a> <a class="dropdown-item" href="/projects/software/">Software</a> </div> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/fun/">fun</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Hesselberth</span> Lab </h1> <p class="desc"></p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/hesselberth-lab-hex-logo-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/hesselberth-lab-hex-logo-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/hesselberth-lab-hex-logo-1400.webp"></source> <img src="/assets/img/hesselberth-lab-hex-logo.jpg" class="img-fluid z-depth-1 rounded" width="auto" height="auto" alt="hesselberth-lab-hex-logo.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <div class="address"> <p><em>Our affiliations</em>: <br> <a href="https://medschool.cuanschutz.edu/rbi" rel="external nofollow noopener" target="_blank">RNA Bioscience Initiative</a> <br> <a href="https://medschool.cuanschutz.edu/biochemistry" rel="external nofollow noopener" target="_blank">Biochemistry &amp; Molecular Genetics</a> <br> <a href="https://www.cuanschutz.edu/" rel="external nofollow noopener" target="_blank">University of Colorado School of Medicine</a></p> </div> </div> <div class="clearfix"> <p>We are a research laboratory of RNA biologists, technology developers, and data analysts focused on discovering and translating fundamental principles of RNA regulation.</p> <p>A major effort in the lab is to <strong>understand how RNA damage and repair are integrated with stress responses</strong> by combining method development, bioinformatics, genetics, biochemistry, and cell biology. We developed <a href="https://pubmed.ncbi.nlm.nih.gov/26001965/" rel="external nofollow noopener" target="_blank">new sequencing approaches</a> to identify damaged RNAs and <a href="https://pubmed.ncbi.nlm.nih.gov/29212664/" rel="external nofollow noopener" target="_blank">new model systems</a> to understand the causes and consequences of RNA damage. Our studies have led to <a href="https://elifesciences.org/articles/42262" rel="external nofollow noopener" target="_blank">new concepts</a> in post-transcriptional regulation with broad implications for understanding the role of RNA damage and repair.</p> <p>A second focus is on <strong>molecular tools that enhance single-cell and spatial transcriptomic experiments</strong>. Our recent advances include the <a href="https://pubmed.ncbi.nlm.nih.gov/32286626/" rel="external nofollow noopener" target="_blank">measurement of enzyme activity and inhibition</a> in single-cell assays and <a href="https://pubmed.ncbi.nlm.nih.gov/33843587/" rel="external nofollow noopener" target="_blank">tracking protein molecules and measuring associated gene expression responses</a> in animal tissue.</p> <p>Our newest efforts focus on <strong>RNA therapeutics and vaccines</strong>. One is a collaboration with clinicians at Children’s Hospital Colorado to develop RNA therapies for pediatric neurological diseases, and a second emphasis is on development of mRNA vaccines that promote unique immune responses.</p> <p><em>See the <strong>projects</strong> link above for more detail on these studies.</em></p> </div> <h2><a href="/publications/" style="color: inherit;" class="font-weight-bold">selected publications</a></h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/white-aa-trna-seq-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/white-aa-trna-seq-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/white-aa-trna-seq-1400.webp"></source> <img src="/assets/img/publication_preview/white-aa-trna-seq.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="white-aa-trna-seq.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="White2024-pt" class="col-sm-8"> <div class="title">Nanopore sequencing of intact aminoacylated tRNAs</div> <div class="author"> Laura K White, Aleksandar Radakovic, Marcin P Sajek, Kezia Dobson, Kent A Riemondy, Samantha Pozo, Jack W Szostak, and Jay R Hesselberth</div> <div class="periodical"> <em>bioRxiv</em>, Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.biorxiv.org/content/10.1101/2024.11.18.623114" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">bioRxiv</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1101/2024.11.18.623114" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Transfer RNAs (tRNA) are decorated during biogenesis with a variety of modifications that modulate their stability, aminoacylation, and decoding potential during translation. The complex landscape of tRNA modification presents significant analysis challenges and to date no single approach enables the simultaneous measurement of important but disparate chemical properties of individual, mature tRNA molecules. We developed a new, integrated approach to analyze the sequence, modification, and aminoacylation state of tRNA molecules in a high throughput nanopore sequencing experiment, leveraging a chemical ligation that embeds the charged amino acid in an adapted tRNA molecule. During nanopore sequencing, the embedded amino acid generates unique distortions in ionic current and translocation speed, enabling application of machine learning approaches to classify charging status and amino acid identity. Specific applications of the method indicate it will be broadly useful for examining relationships and dependencies between tRNA sequence, modification, and aminoacylation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/white-43-mods-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/white-43-mods-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/white-43-mods-1400.webp"></source> <img src="/assets/img/publication_preview/white-43-mods.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="white-43-mods.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="White2024-np" class="col-sm-8"> <div class="title">Comparative analysis of 43 distinct RNA modifications by nanopore tRNA sequencing</div> <div class="author"> Laura K White, Kezia Dobson, Samantha Pozo, Jill M Bilodeaux, Shelby E Andersen, Amber Baldwin, Chloe Barrington, Nadine Koertel, Federico Martinez-Seidel, Saylor M Strugar, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Kristin E.N. Watt, Neelanjan Mukherjee, Jay R Hesselberth' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, ''); ">3 more authors</span> </div> <div class="periodical"> Jul 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.biorxiv.org/content/10.1101/2024.07.23.604651" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">bioRxiv</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1101/2024.07.23.604651" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Transfer RNAs are the fundamental adapter molecules of protein synthesis and the most abundant and heterogeneous class of noncoding RNA molecules in cells. The study of tRNA repertoires remains challenging, complicated by the presence of dozens of post transcriptional modifications. Nanopore sequencing is an emerging technology with promise for both tRNA sequencing and the detection of RNA modifications; however, such studies have been limited by the throughput and accuracy of direct RNA sequencing methods. Moreover, detection of the complete set of tRNA modifications by nanopore sequencing remains challenging. Here we show that recent updates to nanopore direct RNA sequencing chemistry (RNA004) combined with our own optimizations to tRNA sequencing protocols and analysis workflows enable high throughput coverage of tRNA molecules and characterization of nanopore signals produced by 43 distinct RNA modifications. We share best practices and protocols for nanopore sequencing of tRNA and further report successful detection of low abundance mitochondrial and viral tRNAs, providing proof of concept for use of nanopore sequencing to study tRNA populations in the context of infection and organelle biology. This work provides a roadmap to guide future efforts towards de novo detection of RNA modifications across multiple organisms using nanopore sequencing.Competing Interest StatementL.K.W. has received travel and accommodation expenses from Oxford Nanopore Technologies to present at scientific meetings, and has been a participant in ONT Early Access programs for SQK-RNA004. The remaining authors declare no conflicts of interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/sheridan-portable-program-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/sheridan-portable-program-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/sheridan-portable-program-1400.webp"></source> <img src="/assets/img/publication_preview/sheridan-portable-program.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="sheridan-portable-program.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Sheridan2024-ls" class="col-sm-8"> <div class="title">A specific and portable gene expression program underlies antigen archiving by lymphatic endothelial cells</div> <div class="author"> Ryan M Sheridan, Thu A Doan, Cormac Lucas, Tadg S Forward, Aspen Uecker-Martin, Thomas E Morrison, Jay R Hesselberth, and Beth A Jiron Tamburini</div> <div class="periodical"> Apr 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.biorxiv.org/content/10.1101/2024.04.01.587647" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">bioRxiv</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1101/2024.04.01.587647" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Antigens from protein subunit vaccination traffic from the tissue to the draining lymph node, either passively via the lymph or carried by dendritic cells at the local injection site. Lymph node (LN) lymphatic endothelial cells (LEC) actively acquire and archive foreign antigens, and archived antigen can be released during subsequent inflammatory stimulus to improve immune responses. Here, we answer questions about how LECs achieve durable antigen archiving and whether there are transcriptional signatures associated with LECs containing high levels of antigen. We used single cell sequencing in dissociated LN tissue to quantify antigen levels in LEC and dendritic cell populations at multiple timepoints after immunization, and used machine learning to define a unique transcriptional program within archiving LECs that can predict LEC archiving capacity in independent data sets. Finally, we validated this modeling, showing we could predict antigen archiving from a transcriptional dataset of CHIKV infected mice and demonstrated in vivo the accuracy of our prediction. Collectively, our findings establish a unique transcriptional program in LECs that promotes antigen archiving that can be translated to other systems. \#\#\# Competing Interest Statement The authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/rna-repair-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/rna-repair-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/rna-repair-1400.webp"></source> <img src="/assets/img/publication_preview/rna-repair.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="rna-repair.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="White2023-am" class="col-sm-8"> <div class="title">Nanopore sequencing of internal 2'-PO4 modifications installed by RNA repair</div> <div class="author"> Laura K White, Saylor M Strugar, Andrea MacFadden, and Jay R Hesselberth</div> <div class="periodical"> <em>RNA</em>, Jun 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://pubmed.ncbi.nlm.nih.gov/36854608" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PubMed</a> <a href="https://www.biorxiv.org/content/10.1101/2022.05.29.493267" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">bioRxiv</a> <a href="https://github.com/hesselberthlab/RNARePore" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">GitHub</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="36854608" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Ligation by plant and fungal RNA ligases yields an internal 2'-phosphate group on each RNA ligation product. In budding yeast, this covalent mark occurs at the splice junction of two targets of ligation: intron-containing tRNAs and the messenger RNA HAC1 The repertoire of RNA molecules repaired by RNA ligation has not been explored due to a lack of unbiased approaches for identifying RNA ligation products. Here, we define several unique signals produced by 2'-phosphorylated RNAs during nanopore sequencing. A 2'-phosphate at the splice junction of HAC1 mRNA inhibits 5' $\rightarrow$ 3' degradation, enabling detection of decay intermediates in yeast RNA repair mutants by nanopore sequencing. During direct RNA sequencing, intact 2'-phosphorylated RNAs on HAC1 and tRNAs produce diagnostic changes in nanopore current properties and base calling features, including stalls produced as the modified RNA translocates through the nanopore motor protein. These approaches enable directed studies to identify novel RNA repair events in other contexts.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/white-review-nanopore-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/white-review-nanopore-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/white-review-nanopore-1400.webp"></source> <img src="/assets/img/publication_preview/white-review-nanopore.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="white-review-nanopore.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="White2022-sr" class="col-sm-8"> <div class="title">Modification mapping by nanopore sequencing</div> <div class="author"> Laura K White, and Jay R Hesselberth</div> <div class="periodical"> <em>Front. Genet.</em>, Oct 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://pubmed.ncbi.nlm.nih.gov/36386798" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PubMed</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="36386798" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Next generation sequencing (NGS) has provided biologists with an unprecedented view into biological processes and their regulation over the past 2 decades, fueling a wave of development of high throughput methods based on short read DNA and RNA sequencing. For nucleic acid modifications, NGS has been coupled with immunoprecipitation, chemical treatment, enzymatic treatment, and/or the use of reverse transcriptase enzymes with fortuitous activities to enrich for and to identify covalent modifications of RNA and DNA. However, the majority of nucleic acid modifications lack commercial monoclonal antibodies, and mapping techniques that rely on chemical or enzymatic treatments to manipulate modification signatures add additional technical complexities to library preparation. Moreover, such approaches tend to be specific to a single class of RNA or DNA modification, and generate only indirect readouts of modification status. Third generation sequencing technologies such as the commercially available ``long read'' platforms from Pacific Biosciences and Oxford Nanopore Technologies are an attractive alternative for high throughput detection of nucleic acid modifications. While the former can indirectly sense modified nucleotides through changes in the kinetics of reverse transcription reactions, nanopore sequencing can in principle directly detect any nucleic acid modification that produces a signal distortion as the nucleic acid passes through a nanopore sensor embedded within a charged membrane. To date, more than a dozen endogenous DNA and RNA modifications have been interrogated by nanopore sequencing, as well as a number of synthetic nucleic acid modifications used in metabolic labeling, structure probing, and other emerging applications. This review is intended to introduce the reader to nanopore sequencing and key principles underlying its use in direct detection of nucleic acid modifications in unamplified DNA or RNA samples, and outline current approaches for detecting and quantifying nucleic acid modifications by nanopore sequencing. As this technology matures, we anticipate advances in both sequencing chemistry and analysis methods will lead to rapid improvements in the identification and quantification of these epigenetic marks.</p> </div> </div> </div> </li> </ol> </div> <h2 style="color: inherit;" class="font-weight-bold">news</h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row"><span class="badge news-color font-weight-bold text-uppercase align-middle date-ml-3">Aug 26, 2024</span></th> <td> <ul> <li> Welcome to Emily DeCurtis, a rotation student from the Molecular Biology program. </li> <li> Welcome to Grace Li, an undergraduate researcher from UC Denver, and welcome back Sami del Pozo, RBI summer intern. </li> <li> We were awarded an <a href="https://new.nsf.gov/news/nsf-nih-partner-new-research-develop-rna-based" rel="external nofollow noopener" target="_blank">NSF MFB</a> grant with Laura White, Olivia Rissland, and Matt Taliaferro to study <em>“Cracking the Codes: Understanding the Rules of mRNA Localization and Translation”</em>. </li> </ul> </td> </tr> <tr> <th scope="row"><span class="badge news-color font-weight-bold text-uppercase align-middle date-ml-3">Jul 9, 2023</span></th> <td> <ul> <li>Welcome to Christina Akirtava, PhD, who joined the RBI Informatics Fellows, and a new PRA, Kelly McEntee.</li> <li>Ira Fleming was appointed to the Molecular Biology T32 grant.</li> <li>Jay was promoted to Professor of Biochemistry and Molecular Genetics.</li> <li>Welcome to three new students Ira Fleming (MSTP &gt; MOLB), Ashlyn Stahly, (MSTP &gt; MOLB), and Jill Bilodeaux (MICB), a new PRA (Kezia Dobson), and a summer intern (Samantha del Pozo from the <a href="https://medschool.cuanschutz.edu/rbi/training-and-education/summer-internship-program/about" rel="external nofollow noopener" target="_blank">RBI summer internship</a>).</li> <li>Farewell to Saylor Strugar, who is joining the <a href="https://ipib.wisc.edu/" rel="external nofollow noopener" target="_blank">University of Wisconsin</a> for graduate school, Rachel Ancar, MD, PhD, who graduated from the <a href="https://medschool.cuanschutz.edu/mstp" rel="external nofollow noopener" target="_blank">CU MSTP</a> and is moving to Mass General Hospital for residency, Mandy Richer, PhD, who took a position at <a href="https://honeycomb.bio/" rel="external nofollow noopener" target="_blank">Honeycomb Bio</a>, Tyler Matheny, PhD, who took a position at <a href="https://www.sophiagenetics.com/" rel="external nofollow noopener" target="_blank">Sophia Genetics</a>, and Aspen Martin, who took a position with <a href="https://foresight-dx.com/" rel="external nofollow noopener" target="_blank">Foresight Diagnostics</a>.</li> <li>Jay won the <a href="https://research.cuanschutz.edu/university-research/news-events/research-awards-ceremony#ac-research-administration-award-7?utm_source=miragenews&amp;utm_medium=miragenews&amp;utm_campaign=news" rel="external nofollow noopener" target="_blank">CU Anschutz Research Faculty Collaborator Award</a>.</li> <li>We were awarded a <a href="https://www.cancerleague.org/" rel="external nofollow noopener" target="_blank">Cancer League of Colorado</a> collaborative award with Linda van Dyk, PhD to study <em>“Immune targeting therapies to disrupt virus-associated lymphomas.</em>”</li> </ul> </td> </tr> <tr> <th scope="row"><span class="badge news-color font-weight-bold text-uppercase align-middle date-ml-3">Dec 1, 2022</span></th> <td> <ul> <li>Catherine Nicholas was awarded an NIH/NIDDK F31 NRSA.</li> <li>We were awarded an <a href="https://www.cu.edu/ab-nexus/ab-nexus-grant-awardees" rel="external nofollow noopener" target="_blank">AB Nexus</a> collaborative award with Ondrej Kostov and Marvin Caruthers to study <em>“Gene-specific antisense oligonucleotide therapies for diseases of haploinsufficiency.</em>”</li> <li>We were awarded a <a href="https://research.cuanschutz.edu/research-ppg/cu-anschutz-som-programmatic-incubator-for-research-(cu-aspire)-program" rel="external nofollow noopener" target="_blank">CU ASPIRE</a> award with Beth Tamburini, Tem Morrison, and Jenna Guthmiller to study <em>“Mechanisms of immune protection and pathology by lymph node stromal cells.</em>”</li> <li>We were awarded an <a href="https://reporter.nih.gov/search/RzmDYkaaM0qOb3JTwvzbLw/project-details/10508093" rel="external nofollow noopener" target="_blank">NIH/NIAID R21</a> with Ross Kedl to study <em>“mRNA encoding of immune receptor-targeting antibodies for the augmentation of vaccine-elicited cellular immunity.</em>”</li> <li>We won the departmental pumpkin carving competition with a <a href="https://twitter.com/jayhesselberth/status/1581013613036204033" rel="external nofollow noopener" target="_blank">nanopumpkin sequencer</a>, complete with butternut “pore”, acorn “motor”, and a threaded vine “RNA” (truly a masterpiece).</li> </ul> </td> </tr> </table> </div> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2024 Hesselberth Lab. Powered by <a href="http://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Last updated: November 24, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js"></script> <script defer src="/assets/js/common.js"></script> <script defer src="/assets/js/copy_code.js" type="text/javascript"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>