---
layout: page
title: RNA damage and repair
description: How is RNA damage integrated with stress responses?
img: /assets/img/rna-repair.png
importance: 1
category: research
---

Like DNA repair systems that surveil and repair genome damage to preserve
the genetic code, the products of RNA damage are substrates for coupled
end modification and processing steps. But unlike our deep under-standing
of the DNA damage response, we are only beginning to learn how RNA damage,
end modification, and processing are integrated in the RNA damage response
to orchestrate RNA processing for regulatory control.

Our research program tackles open questions in RNA repair by combining
genetics, biochemistry, and bioinformat-ics to explore the causes and
consequences of RNA damage.  In doing so, we have uncovered new concepts
in post-transcriptional regulation and are uniquely positioned to continue
to explore the surprising roles of the RNA damage response in biology.

RNA repair involves three sequential actvities:

1. Spontaneous or enzyme-catalyzed RNA cleavage ("damage").
2. Remodeling of new RNA termini by RNA end modifying enzymes ("healing").
3. Rejoining of the broken ends by an RNA ligase ("sealing").

<img style="float: right;" src="/assets/img/rna-repair.svg">

RNA repair catalyzes rejoining of RNA fragments generated during [tRNA
splicing](#trna-splicing) and a unique mRNA processing event that
regulates the [unfolded protein response](#unfolded-protein-response).
There are hints that RNA repair may play a bigger role in gene expression.
For example, many proteins of unknown function tend to co-evolve with [RNA
repair genes](https://www.ncbi.nlm.nih.gov/pubmed/27536007). And there are
suggestions that [damaged ribosomes can be
repaired](https://www.ncbi.nlm.nih.gov/pubmed/27789694), presumably to
mitigate the cost of building a new ribosome.

We are broadly interested in understanding how cells use RNA repair to
remodel the transcriptome.

### tRNA splicing

<img style="float: right;" src="/assets/img/trna-block.svg">

We [recently generated](https://pubmed.ncbi.nlm.nih.gov/29212664/) budding
yeast cells that lack normally essential compo-nents of RNA repair. RNA
ligase enzymes catalyze rejoining of RNA after cleavage of phosphodiester
linkag-es. RNA ligation in budding yeast is catalyzed by two separate
enzymes that ligate exons of tRNA during splicing and HAC1 mRNA during
activation of the unfolded protein response.  The yeast RNA ligase Trl1
joins 2',3'-cyclic phosphate and 5'-OH RNA fragments 32, yielding a
phosphodiester linkage with a 2'-phosphate that is subsequently removed by
the 2'-phosphotransferase Tpt1. Inspired by previous work in C. elegans,
we rescued the lethality of TRL1 and TPT1 deletions in budding yeast by
expressing intron-less versions of the 10 normally intron-containing
tRNAs, indicating this repair pathway does not have additional essential
functions. These cells are critical reagents that have opened the door to
new studies of eukaryotic RNA repair.

### Unfolded protein response

<img style="float: right;" src="/assets/img/upr.svg">

We [recently discovered](https://pubmed.ncbi.nlm.nih.gov/30874502/) a
previously unappreciated layer of post-transcriptional regulation during
the unfolded protein response (UPR). In the UPR, endoplasmic reticulum
stress activates a large transcriptional program to increase folding
capacity.  In the budding yeast UPR, Ire1 excises an intron from the HAC1
mRNA and the exon products of cleavage are ligated by Trl1, yielding an
mRNA encoding a transcription factor whose translation induces hundreds of
stress-response genes.  Moreover, robust base-pairing between the 5'-exon
and intron inhibits HAC1 translation Using cells with muta-tions in RNA
repair and decay enzymes, we showed that kinase-mediated decay of HAC1
splicing intermedi-ates both suppresses and activates the UPR. Together
with recent studies our findings support a new paradigm in cell signaling:
analogous to kinase-mediated signal transduction through pro-tein
substrates, the UPR is controlled by kinase-mediated signal transduction
through RNA substrates. These studies provide a strong conceptual
foundation for understanding new couplings between RNA damage and repair.

### RNA sequencing approaches to study RNA damage repair

We previously developed methods to capture the [2´,3´-cyclic
phosphate](/publication/2010/Schutz) and [5´-OH](/publication/2015/Peach)
products of RNA decay and have used these broadly to study spontaneous and
enzyme-catalyzed RNA cleavage.

We are conducting targeted and genome-wide searches for other targets of
RNA repair. Such studies could provide new insights into the role of RNA
repair in cellular physiology.

## Funding

This work is funded by an NIH MIRA grant ([R35
GM119550](https://projectreporter.nih.gov/project_info_description.cfm?aid=9322528&icde=35801054&ddparam=&ddvalue=&ddsub=&cr=1&csb=default&cs=ASC&pball=)).
