<!DOCTYPE html> <html lang=""> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>RNA damage and repair | Hesselberth Lab</title> <meta name="author" content="Hesselberth Lab"> <meta name="description" content="Hesselberth Lab at the University of Colorado Asnchutz Medical Campus"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="preconnect" href="https://fonts.googleapis.com"> <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://hesselberthlab.org/projects/rna-repair/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> <script defer data-domain="hesselberthlab.org" src="https://plausible.io/js/script.js"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Hesselberth </span>Lab</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/team/">team</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">projects</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/projects/rna-repair/">RNA damage and repair</a> <a class="dropdown-item" href="/projects/molecular-tech/">Molecular technologies</a> <a class="dropdown-item" href="/projects/rna-therapies/">RNA therapies</a> <a class="dropdown-item" href="/projects/software/">Software</a> </div> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/fun/">fun</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">RNA damage and repair</h1> <p class="post-description"></p> </header> <article> <p>Like DNA repair systems that surveil and repair genome damage to preserve the genetic code, the products of RNA damage are substrates for coupled end modification and processing steps. But unlike our deep understanding of the DNA damage response, we are only beginning to learn how RNA damage, end modification, and processing are integrated in the RNA damage response to orchestrate RNA processing for regulatory control.</p> <p><img style="float: right; max-width: 400px" src="/assets/img/rna-repair.jpg"></p> <p>Growing evidence indicates RNA repair activities can be linked together to create elegant forms of post-transcriptional control. We and others discovered examples of RNA processing by “kinase-mediated decay” wherein RNAs with 5´-OH termini are degraded by sequential 5´-OH phosphorylation and 5´-3´ degradation by a 5´-phosphate-dependent exoribonuclease. Given the diversity of RNA repair activities, other dependencies between RNA end modification and processing could be used for new forms of post-transcriptional regulation.</p> <p>Our research program tackles open questions in RNA repair by combining genetics, biochemistry, and bioinformatics to explore the causes and consequences of RNA damage. In doing so, we have uncovered new concepts in post-transcriptional regulation and are uniquely positioned to continue to explore the surprising roles of the RNA damage response in biology.</p> <h3 id="rna-sequencing-approaches-to-study-rna-damage--repair">RNA sequencing approaches to study RNA damage &amp; repair</h3> <p>We previously developed methods to capture the <a href="https://pubmed.ncbi.nlm.nih.gov/20075163/" rel="external nofollow noopener" target="_blank">2’,3’-cyclic phosphate</a> and <a href="https://pubmed.ncbi.nlm.nih.gov/26001965/" rel="external nofollow noopener" target="_blank">5’-OH</a> products of RNA decay and used them to characterize products of spontaneous and enzyme-catalyzed RNA cleavage.</p> <p><img style="float: right; max-width: 400px" src="/assets/img/rna-repair-nanopore.jpg"></p> <p>Recently we developed an approach leveraging <a href="https://pubmed.ncbi.nlm.nih.gov/36854608/" rel="external nofollow noopener" target="_blank">nanopore sequencing</a> to identify RNA 2´-phosphorylation — a <em>bona fide</em> biochemical mark of RNA repair. This approach enabled unambiguous detection of repair events on tRNA products of ligation and the Hac1 mRNA, which is ligated during the unfolded protein responses. We also identified a collection of mRNAs that generate similar signals during nanopore sequencing, suggesting they are also targets of repair. We continue to apply this approach to characterize RNA repair events during stress conditions.</p> <div class="publications"> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/rna-repair-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/rna-repair-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/rna-repair-1400.webp"></source> <img src="/assets/img/publication_preview/rna-repair.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="rna-repair.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="White2023-am" class="col-sm-8"> <div class="title">Nanopore sequencing of internal 2'-PO4 modifications installed by RNA repair</div> <div class="author"> Laura K White, Saylor M Strugar, Andrea MacFadden, and Jay R Hesselberth</div> <div class="periodical"> <em>RNA</em>, Jun 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://pubmed.ncbi.nlm.nih.gov/36854608" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PubMed</a> <a href="https://www.biorxiv.org/content/10.1101/2022.05.29.493267" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">bioRxiv</a> <a href="https://github.com/hesselberthlab/RNARePore" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">GitHub</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="36854608" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Ligation by plant and fungal RNA ligases yields an internal 2'-phosphate group on each RNA ligation product. In budding yeast, this covalent mark occurs at the splice junction of two targets of ligation: intron-containing tRNAs and the messenger RNA HAC1 The repertoire of RNA molecules repaired by RNA ligation has not been explored due to a lack of unbiased approaches for identifying RNA ligation products. Here, we define several unique signals produced by 2'-phosphorylated RNAs during nanopore sequencing. A 2'-phosphate at the splice junction of HAC1 mRNA inhibits 5' $\rightarrow$ 3' degradation, enabling detection of decay intermediates in yeast RNA repair mutants by nanopore sequencing. During direct RNA sequencing, intact 2'-phosphorylated RNAs on HAC1 and tRNAs produce diagnostic changes in nanopore current properties and base calling features, including stalls produced as the modified RNA translocates through the nanopore motor protein. These approaches enable directed studies to identify novel RNA repair events in other contexts.</p> </div> </div> </div> </li></ol> </div> <h3 id="trna-splicing">tRNA splicing</h3> <p>We <a href="https://pubmed.ncbi.nlm.nih.gov/29212664/" rel="external nofollow noopener" target="_blank">recently generated</a> budding yeast cells that lack normally essential compo-nents of RNA repair. RNA ligase enzymes catalyze rejoining of RNA after cleavage of phosphodiester linkag-es. RNA ligation in budding yeast is catalyzed by two separate enzymes that ligate exons of tRNA during splicing and HAC1 mRNA during activation of the unfolded protein response. The yeast RNA ligase Trl1 joins 2’,3’-cyclic phosphate and 5’-OH RNA fragments 32, yielding a phosphodiester linkage with a 2’-phosphate that is subsequently removed by the 2’-phosphotransferase Tpt1. Inspired by previous work in <em>C. elegans</em>, we rescued the lethality of <em>TRL1</em> and <em>TPT1</em> deletions in budding yeast by expressing intron-less versions of the 10 normally intron-containing tRNAs, indicating this repair pathway does not have additional essential functions. These cells are critical reagents that have opened the door to new studies of eukaryotic RNA repair. We are currently expanding this approach to study the breadth of RNA repair across evolution.</p> <p>It is noteworty that this approach was unable to bypass the essentialty of the TSEN tRNA splicing complex, resonating with <a href="https://pubmed.ncbi.nlm.nih.gov/22391451/" rel="external nofollow noopener" target="_blank">earlier work</a> from the Hopper lab. We are using the genetic bypass strategy to identify other essential functions of TSEN, which may shed light on its role in the human neurodegenerative disease pontocerebellar hypoplasia.</p> <div class="publications"> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/trna-block-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/trna-block-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/trna-block-1400.webp"></source> <img src="/assets/img/publication_preview/trna-block.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="trna-block.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Cherry2018-cs" class="col-sm-8"> <div class="title">Genetic bypass of essential {RNA} repair enzymes in budding yeast</div> <div class="author"> Patrick D Cherry, Laura K White, Kerri York, and Jay R Hesselberth</div> <div class="periodical"> <em>RNA</em>, Mar 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://pubmed.ncbi.nlm.nih.gov/29212664" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PubMed</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="29212664" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>RNA repair enzymes catalyze rejoining of an RNA molecule after cleavage of phosphodiester linkages. RNA repair in budding yeast is catalyzed by two separate enzymes that process tRNA exons during their splicing andHAC1mRNA exons during activation of the unfolded protein response (UPR). The RNA ligase Trl1 joins 2',3'-cyclic phosphate and 5'-hydroxyl RNA fragments, creating a phosphodiester linkage with a 2'-phosphate at the junction. The 2'-phosphate is removed by the 2'-phosphotransferase Tpt1. We bypassed the essential functions ofTRL1andTPT1in budding yeast by expressing ``prespliced,'' intronless versions of the 10 normally intron-containing tRNAs, indicating this repair pathway does not have additional essential functions. Consistent with previous studies, expression of intronless tRNAs failed to rescue the growth of cells with deletions in components of the SEN complex, implying an additional essential role for the splicing endonuclease. Thetrl1$\Delta$ andtpt1$\Delta$ mutants accumulate tRNA andHAC1splicing intermediates indicative of RNA repair defects and are hypersensitive to drugs that inhibit translation. Failure to induce the unfolded protein response intrl1$\Delta$ cells grown with tunicamycin is lethal owing to their inability to ligateHAC1after its cleavage by Ire1. In contrast,tpt1$\Delta$ mutants grow in the presence of tunicamycin despite reduced accumulation of splicedHAC1mRNA. We optimized a PCR-based method to detect RNA 2'-phosphate modifications and show they are present on ligatedHAC1mRNA. These RNA repair mutants enable new studies of the role of RNA repair in cellular physiology.</p> </div> </div> </div> </li></ol> </div> <h3 id="unfolded-protein-response">Unfolded protein response</h3> <p>We <a href="https://pubmed.ncbi.nlm.nih.gov/30874502/" rel="external nofollow noopener" target="_blank">recently discovered</a> a previously unappreciated layer of post-transcriptional regulation during the unfolded protein response (UPR). In the UPR, endoplasmic reticulum stress activates a large transcriptional program to increase folding capacity. In the budding yeast UPR, Ire1 excises an intron from the HAC1 mRNA and the exon products of cleavage are ligated by Trl1, yielding an mRNA encoding a transcription factor whose translation induces hundreds of stress-response genes. Moreover, robust base-pairing between the 5’-exon and intron inhibits HAC1 translation Using cells with muta-tions in RNA repair and decay enzymes, we showed that kinase-mediated decay of HAC1 splicing intermedi-ates both suppresses and activates the UPR. Together with recent studies our findings support a new paradigm in cell signaling: analogous to kinase-mediated signal transduction through pro-tein substrates, the UPR is controlled by kinase-mediated signal transduction through RNA substrates. These studies provide a strong conceptual foundation for understanding new couplings between RNA damage and repair.</p> <div class="publications"> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/upr-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/upr-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/upr-1400.webp"></source> <img src="/assets/img/publication_preview/upr.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="upr.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Cherry2019-uf" class="col-sm-8"> <div class="title">Multiple decay events target {HAC1} {mRNA} during splicing to regulate the unfolded protein response</div> <div class="author"> Patrick D Cherry, Sally E Peach, and Jay R Hesselberth</div> <div class="periodical"> <em>Elife</em>, Mar 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://pubmed.ncbi.nlm.nih.gov/30874502" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PubMed</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="30874502" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>In the unfolded protein response (UPR), stress in the endoplasmic reticulum (ER) activates a large transcriptional program to increase ER folding capacity. During the budding yeast UPR, Ire1 excises an intron from the HAC1 mRNA and the exon products of cleavage are ligated, and the translated protein induces hundreds of stress-response genes. Using cells with mutations in RNA repair and decay enzymes, we show that phosphorylation of two different HAC1 splicing intermediates is required for their degradation by the 5'$\rightarrow$3' exonuclease Xrn1 to enact opposing effects on the UPR. We also found that ligated but 2'-phosphorylated HAC1 mRNA is cleaved, yielding a decay intermediate with both 5'- and 2'-phosphates at its 5'-end that inhibit 5'$\rightarrow$3' decay and suggesting that Ire1 degrades incompletely processed HAC1. These decay events expand the scope of RNA-based regulation in the budding yeast UPR and have implications for the control of the metazoan UPR.</p> </div> </div> </div> </li></ol> </div> <h2 id="funding">Funding</h2> <ul> <li><a href="https://reporter.nih.gov/search/AWsb18pUaEe-O6bNwTf9KA/project-details/10193187" rel="external nofollow noopener" target="_blank">NIH/NIGMS MIRA</a></li> </ul> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2023 Hesselberth Lab. Powered by <a href="http://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Last updated: June 01, 2023. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js"></script> <script defer src="/assets/js/common.js"></script> <script defer src="/assets/js/copy_code.js" type="text/javascript"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>