<!DOCTYPE html> <html lang=""> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>RNA damage and repair | Hesselberth Lab</title> <meta name="author" content="Hesselberth Lab"> <meta name="description" content="Hesselberth Lab at the University of Colorado Asnchutz Medical Campus"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://hesselberthlab.org/projects/rna-repair/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> <script defer data-domain="hesselberthlab.org" src="https://plausible.io/js/script.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Hesselberth </span>Lab</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people</a> </li> <li class="nav-item "> <a class="nav-link" href="/positions/">positions</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">projects</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/projects/rna-repair/">RNA damage and repair</a> <a class="dropdown-item" href="/projects/molecular-tech/">Molecular technologies</a> </div> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">RNA damage and repair</h1> <p class="post-description"></p> </header> <article> <p>Like DNA repair systems that surveil and repair genome damage to preserve the genetic code, the products of RNA damage are substrates for coupled end modification and processing steps. But unlike our deep understanding of the DNA damage response, we are only beginning to learn how RNA damage, end modification, and processing are integrated in the RNA damage response to orchestrate RNA processing for regulatory control.</p> <p><img style="float: right; max-width: 400px" src="/assets/img/rna-repair.svg"></p> <p>Growing evidence indicates RNA repair activities can be linked together to create elegant forms of post-transcriptional control. We and others discovered examples of RNA processing by “kinase-mediated decay” wherein RNAs with 5’-OH termini are degraded by sequential 5’-OH phosphorylation and 5’-3’ degradation by a 5’-phosphate-dependent exoribonuclease. Given the diversity of RNA repair activities, other dependencies between RNA end modification and processing could be used for new forms of post-transcriptional regulation.</p> <p>Our research program tackles open questions in RNA repair by combining genetics, biochemistry, and bioinformatics to explore the causes and consequences of RNA damage. In doing so, we have uncovered new concepts in post-transcriptional regulation and are uniquely positioned to continue to explore the surprising roles of the RNA damage response in biology.</p> <h3 id="rna-sequencing-approaches-to-study-rna-damage-repair">RNA sequencing approaches to study RNA damage repair</h3> <p>We previously developed methods to capture the <a href="https://pubmed.ncbi.nlm.nih.gov/20075163/" rel="external nofollow noopener" target="_blank">2’,3’-cyclic phosphate</a> and <a href="https://pubmed.ncbi.nlm.nih.gov/26001965/" rel="external nofollow noopener" target="_blank">5’-OH</a> products of RNA decay and have used these broadly to study spontaneous and enzyme-catalyzed RNA cleavage.</p> <p>Recently we developed an approach leveraging <a href="https://www.biorxiv.org/content/10.1101/2022.05.29.493267v1" rel="external nofollow noopener" target="_blank">nanopore sequencing</a> to identify RNA 2´-phosphorylation — a clear biochemical mark of RNA repair.</p> <h3 id="trna-splicing">tRNA splicing</h3> <p><img style="float: right; max-width: 400px" src="/assets/img/trna-block.svg"></p> <p>We <a href="https://pubmed.ncbi.nlm.nih.gov/29212664/" rel="external nofollow noopener" target="_blank">recently generated</a> budding yeast cells that lack normally essential compo-nents of RNA repair. RNA ligase enzymes catalyze rejoining of RNA after cleavage of phosphodiester linkag-es. RNA ligation in budding yeast is catalyzed by two separate enzymes that ligate exons of tRNA during splicing and HAC1 mRNA during activation of the unfolded protein response. The yeast RNA ligase Trl1 joins 2’,3’-cyclic phosphate and 5’-OH RNA fragments 32, yielding a phosphodiester linkage with a 2’-phosphate that is subsequently removed by the 2’-phosphotransferase Tpt1. Inspired by previous work in <em>C. elegans</em>, we rescued the lethality of <em>TRL1</em> and <em>TPT1</em> deletions in budding yeast by expressing intron-less versions of the 10 normally intron-containing tRNAs, indicating this repair pathway does not have additional essential functions. These cells are critical reagents that have opened the door to new studies of eukaryotic RNA repair. We are currently expanding this approach to study the breadth of RNA repair across evolution.</p> <h3 id="unfolded-protein-response">Unfolded protein response</h3> <p><img style="float: right; max-width: 200px" src="/assets/img/upr.jpg"></p> <p>We <a href="https://pubmed.ncbi.nlm.nih.gov/30874502/" rel="external nofollow noopener" target="_blank">recently discovered</a> a previously unappreciated layer of post-transcriptional regulation during the unfolded protein response (UPR). In the UPR, endoplasmic reticulum stress activates a large transcriptional program to increase folding capacity. In the budding yeast UPR, Ire1 excises an intron from the HAC1 mRNA and the exon products of cleavage are ligated by Trl1, yielding an mRNA encoding a transcription factor whose translation induces hundreds of stress-response genes. Moreover, robust base-pairing between the 5’-exon and intron inhibits HAC1 translation Using cells with muta-tions in RNA repair and decay enzymes, we showed that kinase-mediated decay of HAC1 splicing intermedi-ates both suppresses and activates the UPR. Together with recent studies our findings support a new paradigm in cell signaling: analogous to kinase-mediated signal transduction through pro-tein substrates, the UPR is controlled by kinase-mediated signal transduction through RNA substrates. These studies provide a strong conceptual foundation for understanding new couplings between RNA damage and repair.</p> <h2 id="funding">Funding</h2> <ul> <li><a href="https://reporter.nih.gov/search/AWsb18pUaEe-O6bNwTf9KA/project-details/10193187" rel="external nofollow noopener" target="_blank">NIH/NIGMS MIRA</a></li> </ul> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Hesselberth Lab. Powered by <a href="http://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Last updated: May 29, 2023. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js"></script> <script defer src="/assets/js/common.js"></script> <script defer src="/assets/js/copy_code.js" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>